OpenOversight/tests/routes/test_descriptions.py

```diff
@@ -124,7 +124,8 @@ def test_admins_can_edit_descriptions(mockdata, client, session):
         description = Description(
             text_contents=old_description,
             officer_id=officer.id,
-            updated_at=original_date,
+            created_at=original_date,
+            last_updated_at=original_date,
```

Comment:
I don't think we need the `last_updated_at` spot filled in unless it is an editing action, but we do need the `last_updated_by` column filled in. `created_at` doesn't seem necessary in general unless you'd like to use it in a assertion statement within the test, imo.